stream-compositions/services/cursor-service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,27 @@
+server:
+  port: 9915
+
+# API Registry client configuration
+eureka:
+  instance:
+    metadata-map:
+      public: true
```

Comment:
I think this was used by gateway to determine whether to expose service - does not seem appropriate here.